agent/firewall.go

```diff
@@ -110,24 +110,27 @@ func (fw *Firewall) Init(netif NetIf) error {
 	// Default permissive rules to allow ssh/ICMP
 	// from the Host to the Endpoint. Normally wildcard U32 filter would
 	// block all the Host to Endpoint traffic.
-	inputRules := new(firewallRules)
-	inputRules.Add(fmt.Sprintf("-d %s", fw.Agent.networkConfig.currentHostIP))
-	inputRules.Add(fmt.Sprintf("-d %s/%d", fw.Agent.networkConfig.currentHostGW, fw.Agent.networkConfig.currentHostGWNetSize))
-	inputRules.Add("-p udp --sport 68 --dport 67 -d 255.255.255.255")
-	inputRules.Add(fmt.Sprintf("-p tcp -m tcp --sport 22 -d %s", fw.Agent.networkConfig.currentHostIP))
-
-	outputRules := new(firewallRules)
-	outputRules.Add(fmt.Sprintf("-d %s", fw.Agent.networkConfig.currentHostIP))
-	outputRules.Add(fmt.Sprintf("-d %s/%d", fw.Agent.networkConfig.currentHostGW, fw.Agent.networkConfig.currentHostGWNetSize))
-	outputRules.Add(fmt.Sprintf("-p tcp -m tcp --sport 22 -d %s", fw.Agent.networkConfig.currentHostIP))
-
-	forwardRules := new(firewallRules)
-
-	c1 := NewFirewallChain("INPUT", []string{"i"}, *inputRules, fw.prepareChainName("INPUT"))
-	c2 := NewFirewallChain("OUTPUT", []string{"o"}, *outputRules, fw.prepareChainName("OUTPUT"))
-	c3 := NewFirewallChain("FORWARD", []string{"i", "o"}, *forwardRules, fw.prepareChainName("FORWARD"))
-	chains := [numberOfDefaultChains]FirewallChain{*c1, *c2, *c3}
-	fw.chains = chains
+	// Allow ICMP, DHCP and SSH between host and instances.
+	hostAddr := fw.Agent.networkConfig.romanaGW
```

Comment:
It is one of the host's addresses -- IP of romana-gw interface -- and it's the one we're operating the iptables rules around. It's locally scoped, so it is clear in the context.